### PR TITLE
feat(octavia): added l7 policy edition page

### PR DESCRIPTION
**Pull request by JacquesLarique**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/octavia-policies-and-rules`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-12698
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ (n/a)

## Description

Added the L7 Policy edition page

### CODE REVIEW

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/create/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
   "octavia_load_balancer_create_l7_policy_title": "L7 Policy",
-  "octavia_load_balancer_create_l7_policy_description": "Une L7 policy est l'association entre une ou plusieurs L7 rules et un listener. Son attribut principal est l'action qui est faite lorsque toutes les L7 rules associées retourne vrai."
+  "octavia_load_balancer_create_l7_policy_description": "Une L7 policy est l'association entre une ou plusieurs L7 rules et un listener. Son attribut principal est l'action qui est faite lorsque toutes les L7 rules associées retourne vrai.",
```
Comment (dectotam) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/create/translations/Messages_fr_FR.json:3]:
```suggestion
  "octavia_load_balancer_create_l7_policy_description": "Une L7 policy est l'association entre une ou plusieurs L7 rules et un listener. Son attribut principal est l'action qui est faite lorsque toutes les L7 rules associées retournent vrai.",
```

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/edit/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "octavia_load_balancer_edit_l7_policy_title": "L7 Policy",
+  "octavia_load_balancer_edit_l7_policy_description": "Une L7 policy est l'association entre une ou plusieurs L7 rules et un listener. Son attribut principal est l'action qui est faite lorsque toutes les L7 rules associées retourne vrai."
```
Comment (dectotam) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/edit/translations/Messages_fr_FR.json:3]:
```suggestion
  "octavia_load_balancer_edit_l7_policy_description": "Une L7 policy est l'association entre une ou plusieurs L7 rules et un listener. Son attribut principal est l'action qui est faite lorsque toutes les L7 rules associées retournent vrai."
```

**Review by dectotam — CHANGES_REQUESTED** [review]

**Review by qpavy — APPROVED** [review]

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/components/policy-form/controller.js [hunk 3]
```diff
@@ -2,13 +2,23 @@ import { ACTIONS } from '../../constants';
...
     this.URL_PLACEHOLDER = URL_PLACEHOLDER;
     this.actions = ACTIONS_LIST;
   }
 
+  $onInit() {
+    this.submitLabel = this.submitLabel
+      ? this.submitLabel
+      : this.$translate.instant(
+          'octavia_load_balancer_create_l7_policy_submit',
+        );
```
Comment (darsene) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/components/policy-form/controller.js:19]: Sounds more logic to store the value in a new property when it's possible to override a default value.
Moreover, the given value `submitLabel` has to be translated as well. 
```suggestion
    this.submitBtnLabel = this.$translate.instant(
           this.submitLabel ?? 'octavia_load_balancer_create_l7_policy_submit',
        );
```

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/components/policy-form/template.html [hunk 4]
```diff
@@ -127,7 +127,7 @@
     </oui-field>
     <oui-form-actions
         data-disabled="!policy.$valid"
-        data-submit-text="{{:: 'octavia_load_balancer_create_l7_policy_submit' | translate }}"
+        data-submit-text="{{:: $ctrl.submitLabel }}"
```
Comment (darsene) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/components/policy-form/template.html:130]:
```suggestion
        data-submit-text="{{:: $ctrl.submitBtnLabel }}"
```
or
```suggestion
        data-submit-text="{{:: ($ctrl.submitLabel || 'octavia_load_balancer_create_l7_policy_submit') | translate }}"
```

**Review by darsene — APPROVED** [review]

**Review by dectotam — APPROVED** [review]

**Pull request merged**